main.go

```diff
@@ -245,6 +245,8 @@ func main() {
 
 	log.Printf("wrote %s", certFile)
 
+	writeKeystore(certDir, key, certificate)
```

Comment:
Is it OK to assume Java KeyStore and the PEM Cert go to the same directory? Is that how we configure our JRE for Platform servers?

Reply:
Platform defaults to reading from etc under the install directory ("/service" for Docker). That won't work here, as we need it to go into a volume. So we'll probably have to configure the Platform service to read from wherever certificate-init-container puts the keystore. We could do that configuration change in the application code.